### DISCUSSION TITLE
Split of mp3 files requires at least 3 minutes play time

### DISCUSSION THREAD

**Discussion by mark-wehrmann** [audiamus/AaxAudioConverter#217] [Q&A]
Dear Audiamus

I changed the split to 1 minute but AaxAudioConverter seems to switch it back to a minimum of 3 minutes.

The reason why I need only 1 minute is as follows:
I use your tool to transfer an Audible Audio Book to an underwater headset for swimming which has an integrated MP3-Player (to overcome limitations of Bluetooth under water). However that intergrated player is rather rudimentary and only allows to switch single tracks back- and forward, also there is of cause no display etc. Unfortunateley there is even no option to just move within a track a few seconds which is normally required to repeat when there was noise around or your mind swifted away etc.

My workaround would be to create small parts within the chapters. That way I could utilize the track switching button of the device to repeat a maximum of one minute. However repeating 3 minutes would be definitely too much.

Any specific suggestion why this value you couldn't be less than 3 minutes. I would even enjoy to have very small parts up to 15 seconds or so.

Thanks for damned good application and suppport

M

**Comment by audiamus** [comment]
> Any specific suggestion why this value you couldn't be less than 3 minutes

One reason is bit of user guidance, to avoid unwanted flooding of the output folder. But there is also a technical one. Actual track length depends on silence availability. And that is more critical the shorter the desired track length is, up to a point where splitting doesn't make sense. With the current hard-coded 3 minutes minimum I did not encounter issues here, but testing with shorter tracks I remember having some.

**Comment by Osiloko12** [comment]
Ok thanks for letting me know how
